src/data_info/routes/schemas/users.clj

```diff
@@ -0,0 +1,11 @@
+(ns data-info.routes.schemas.users
+  (:use [common-swagger-api.schema :only [describe
+                                          NonBlankString
+                                          StandardUserQueryParams]]
+        [data-info.routes.schemas.common])
+  (:require [schema.core :as s]))
+
+(s/defschema QualifiedUser (describe NonBlankString "A qualified username like foo#bar"))
```

Comment:
We don't have to use `defschema` for this parameter. Usually I'll just use `def` for simple (non-map/body/request) parameters, or for parts of another schema (see `data-info.routes.schemas.common`). But it should also be fine to use it here.

I don't think the documentation from this `describe` will display in the swagger docs, so maybe it should be used as the documentation for `:user` in the `UserGroupsReturn` schema?